packages/common/src/components/Input/index.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import Input from './';
+import { ThemeDecorator } from '../../stories/decorators';
+
+const stories = storiesOf('components/Input', module).addDecorator(
+  ThemeDecorator
+);
+
+stories.add('Basic Input', () => <Input />);
+stories.add('Block Input', () => <Input />);
```

Comment:
You can just chain all `add` calls

```suggestion
    .add('Block Input', () => <Input />);
```

Reply:
Done in all! Thank you!